CliFx.Tests/CliApplicationBuilderTests.cs

```diff
@@ -1,8 +1,9 @@
-﻿using System;
-using System.IO;
-using CliFx.Services;
+﻿using CliFx.Services;
+using CliFx.Tests.Stubs;
 using CliFx.Tests.TestCommands;
 using NUnit.Framework;
+using System;
```

Comment:
Please keep the previous namespace ordering: System. first (here and other places)

Reply:
Visual Studio format document 😑 I'll fix it